mozilla/kitsune
# [bug 1004162] Remove jquery.selectbox.

To verify:

1- Run tests (which includes the js qunit tests)
2- Load an article and verify that there are no js errors and that changing the showfor selection doesn't trigger any

r?
